src/pages/guide/language/index.md

```diff
@@ -15,11 +15,11 @@ Reason is an umbrella project that provides a curated layer for [OCaml](http://o
 
 #### Worry-free
 
-Reason leverages OCaml's language design and type system to give immediate feedback in the form of compile-time errors and avoided anti-patterns.
+Reason leverages OCaml's language design and type system to give immediate feedback in the form of compile-time errors and language features that make it easy to avoid anti-patterns.
```

Comment:
"avoided anti-patterns" was ambiguous to me, so I made it clear that it's the language itself that makes it easy to avoid anti-patterns.